hotsos/core/plugins/system/system.py

```diff
@@ -81,6 +81,10 @@ def date(self):
     def hostname(self):
         return CLIHelper().hostname()
 
+    @cached_property
```

Comment:
I implemented https://github.com/canonical/hotsos/blob/main/hotsos/core/host_helpers/uptime.py some time back partly for this purpose but never got around to adding it to the summary. Personally I think it would be preferable if the uptime was just the elapsed time and not the loadavg etc since that is already in the system summary under ```load:```

Reply:
ack, working on it

Reply:
.... done.

Reply:
you've made me realise how unclear my UptimeHelper is so ive gone ahead and proposed a cleanup in https://github.com/canonical/hotsos/pull/703 which is similar to what you have done here but provides a way to repr(helper) to get the string format. Let me know what you think.

Reply:
#703 is way clearer than what I did here, I approved it and I'll revise this MP after it's merged.

Reply:
@dosaboy, rebased and ready for review